pandas/tests/frame/methods/test_isin.py

```diff
@@ -92,7 +96,8 @@ def test_isin_df_dupe_values(self):
         df1 = DataFrame({"A": [1, 2, 3, 4], "B": [2, np.nan, 4, 4]})
         # just cols duped
         df2 = DataFrame([[0, 2], [12, 4], [2, np.nan], [4, 5]], columns=["B", "B"])
-        with pytest.raises(ValueError):
+        msg = "cannot compute isin with a duplicate axis."
```

Comment:
```suggestion
        msg = r"cannot compute isin with a duplicate axis\."
```

I think meant to escape this